app/__init__.py

```diff
@@ -497,3 +497,26 @@ def connect_handler_notif_page():
         socketio.run(current_app)
     else:
         current_app.run()
+
+
+# Implementation for i18n
+
+from flask.ext.babel import Babel
+from config import LANGUAGES
```

Comment:
Move the import statement to the top of the file along with all the other imports